.github/workflows/tests.yml

```diff
@@ -10,309 +10,25 @@ permissions:
   contents: read
 
 jobs:
-  tests:
```

Comment:
yeah just speeding the feedback loop.

Reply:
Just trying to fix the brew release after the merge today.

Reply:
While you're here, @zabil, why does the automatic rebase action failing on master and how can I fix it?

Reply:
I am not very sure. I don't recollect setting it up. 
I think instead of an automatic rebase we should turn on merge queues https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request-with-a-merge-queue

Reply:
I agree. I don't thing we should be auto rebasing PRs ourselves. I have enabled the following which would force people to update their PRs before we can merge.

<img width="433" alt="image" src="https://github.com/getgauge/gauge/assets/13623227/42ae2824-0eec-43bc-92aa-9384fe14d6d5">

Happy for me to remove the auto rebase action?